src/main.c

```diff
@@ -320,8 +320,8 @@ void app_main(void) {
 
 static void init_coin_config(chain_config_t *coin_config) {
     memset(coin_config, 0, sizeof(chain_config_t));
-    strcpy(coin_config->coinName, CHAINID_COINNAME);
-    coin_config->chainId = CHAIN_ID;
+    strcpy(coin_config->coinName, APP_TICKER);
```

Comment:
Should we also change the field `coinName` by something like `coinTicker`?

Reply:
We could, but in a later PR. :+1: